voidcrew/modules/overmap/code/game/turfs/closed/overmap.dm

```diff
@@ -1,7 +1,6 @@
 /turf/closed/overmap_edge
 	icon = 'voidcrew/modules/overmap/icons/turf/overmap.dmi'
 	icon_state = "overmap"
-	space_lit = TRUE
```

Comment:
if you're removing this from here why not `/turf/open/overmap` too :)

Reply:
ok sure